fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -614,7 +613,11 @@ Function getDecimalV3Function(FunctionCallExpr node, Type[] argumentTypes) {
                 Type returnType = fn.getReturnType();
                 // Decimal v3 function return type maybe need change
                 if (returnType.isDecimalV3() && commonType.isValid()) {
-                    returnType = commonType;
+                    if (FunctionSet.TRUNCATE_DECIMAL.equalsIgnoreCase(fnName)) {
```

Comment:
should comment why TRUNCATE_DECIMAL is special.

Reply:
ok